### ISSUE TITLE
GitHub token is not read from paths

### ISSUE THREAD

**Issue by amalmurali47** [mattduck/gh2md#19]
Running gh2md with the GitHub token in `~/.config/gh2md/token` produces the following:

```
[2021-07-17 13:09:24,333] [INFO] Looking for token in file: /home/user/.config/gh2md/token
[2021-07-17 13:09:24,333] [INFO] Using token from file: /home/user/.config/gh2md/token
[2021-07-17 13:09:24,333] [INFO] Looking for token in file: /home/user/.github-token
[2021-07-17 13:09:24,333] [WARNING] No token found. Access to private repositories will fail
```

The token is resolved in the following (in order):
- A `GITHUB_ACCESS_TOKEN` environment variable.
- `~/.config/gh2md/token`
- `~/.github-token`

The problem is that, when the token is read from either of the files, its value is not returned outside to the calling function. Here's the relevant piece of code:

```python
def get_environment_token():
    try:
        token = os.environ[ENV_GITHUB_TOKEN]
        logger.info("Using token from environment")
        return token
    except KeyError:
        for path in GITHUB_ACCESS_TOKEN_PATHS:
            logger.info(f"Looking for token in file: {path}")
            if os.path.exists(path):
                logger.info(f"Using token from file: {path}")
                with open(path, "r") as f:
                    token = f.read().strip()
```

The fix would be to return the `token` after `token = f.read().strip()`.

**Comment by mattduck** [comment]
Thanks for this - have merged the associated PR and released a new version 1.0.4.

**Comment by amalmurali47** [comment]
@mattduck Awesome, thank you!
